.travis.yml

```diff
@@ -2,19 +2,15 @@
 sudo: false
 language: ruby
 cache: bundler
-env:
+env: 
 rvm:
-  - jruby-1.7.25
+- jruby-1.7.25
```

Comment:
We actually still need 3 build, if you look at the job description:

![screenshot 2017-06-13 08 36 23](https://user-images.githubusercontent.com/640/27082615-80813c5c-5013-11e7-827b-ec22b2055e2a.png)

The first job is;

- Test with JRuby 1.7.25 + released gems on rubyges, this build translate into the Logstash 5.4.1 release (current released stable).
- Test with JRuby 1.7.25 with the 5.x branch, which is the next 5.4.x release.
- The last test with master with JRuby 9k.